pytest/lib/mocknet.py

```diff
@@ -722,12 +691,10 @@ def create_genesis_file(validator_node_names,
 
     total_supply = 0
     for record in genesis_config['records']:
-        if 'Account' in record:
-            if 'account' in record['Account']:
-                if 'locked' in record['Account']['account']:
-                    total_supply += int(record['Account']['account']['locked'])
-                if 'amount' in record['Account']['account']:
-                    total_supply += int(record['Account']['account']['amount'])
+        total_supply += int(
+            record.get('Account', {}).get('account', {}).get('locked', str(0)))
+        total_supply += int(
+            record.get('Account', {}).get('account', {}).get('amount', str(0)))
```

Comment:
This would read better with a `account = record.get('Account', {}).get('account', {})` temporary.